indicators/sledgehammerdiscordbot.yml

```diff
@@ -0,0 +1,32 @@
+title: sledgehammerfakeverification
+description: |
+  Tries to detect the fake sledgehammer login to verify or bookmark verification pages.
+
+level: likely_malicious
+
+references:
+    -https://urlscan.io/result/4b34b0a1-c66c-4228-9d67-c9790dadccb8/
+    -https://urlscan.io/result/46166857-4a6b-48c7-8913-292a1d01ce99/
```

Comment:
I believe this reference would also match the existing rule [bookmark-grabber-d7eb986c](https://github.com/phish-report/IOK/blob/61e3415d93e3055a3cd0513a7d405a19f1757e0c/indicators/bookmark-grabber-d7eb986c.yml)